exchanges/order/timeinforce_test.go

```diff
@@ -9,6 +9,27 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/encoding/json"
 )
 
+func TestTimeInForceIs(t *testing.T) {
+	t.Parallel()
+	tifValuesMap := map[TimeInForce][]TimeInForce{
+		GoodTillCancel | PostOnly:   {GoodTillCancel, PostOnly},
+		GoodTillCancel:              {GoodTillCancel},
+		GoodTillCrossing | PostOnly: {GoodTillCrossing, PostOnly},
+		GoodTillDay:                 {GoodTillDay},
+		GoodTillTime:                {GoodTillTime},
+		GoodTillTime | PostOnly:     {GoodTillTime, PostOnly},
+		ImmediateOrCancel:           {ImmediateOrCancel},
+		FillOrKill:                  {FillOrKill},
+		PostOnly:                    {PostOnly},
+		GoodTillCrossing:            {GoodTillCrossing},
+	}
+	for tif := range tifValuesMap {
+		for _, v := range tifValuesMap[tif] {
+			require.True(t, tif.Is(v))
```

Comment:
🐒  If this fails it's going to be hard to decipher what failed. I'm assuming `tif` will call `.String` but I could be wrong, and if so then use `.String()`
```suggestion
			require.Truef(t, tif.Is(v), "%s should be %s", tif, v)
```